docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md

```diff
@@ -16,7 +16,7 @@ The new `ListOrders` resource was automatically added to all authentications ([r
 
```

Comment:
O trecho que fala "the new resource was automatically added" talvez fique um pouco confuso, considerando que a mudança é só dia 15, não?

Reply:
Faz todo sentido, alterei o texto em vários trechos.